src/scenes/ConferenceList/ConferenceList.module.scss

```diff
@@ -88,3 +88,16 @@
     }
   }
 }
+
+.searchAndDate {
+  display: flex;
+  justify-content: space-between;
+  gap: spacing(loose);
+}
+
+.dateRangePicker {
+  display: flex;
+  width: 400px;
+  padding: 2px;
```

Comment:
This padding breaks the vertical alignment, see:
<img width="686" alt="image" src="https://github.com/user-attachments/assets/05a7805c-3fa6-4bfa-a4f6-749807638e4e">